components/filter/filter-dimension-set-date-time-range-value.js

```diff
@@ -144,14 +161,46 @@ class FilterDimensionSetDateTimeRangeValue extends LocalizeCoreElement(LitElemen
 	async _handleDateChange(e) {
 		if (this.type === 'date') {
 			this.startValue = e.target.startValue ? getUTCDateTimeFromLocalDateTime(e.target.startValue, '0:0') : undefined;
-			this.endValue = e.target.endValue ? getUTCDateTimeFromLocalDateTime(e.target.endValue, '0:0') : undefined;
+			this.endValue = e.target.endValue ? getUTCDateTimeFromLocalDateTime(e.target.endValue, '23:59:59') : undefined;
```

Comment:
This isn't necessarily related to tags specifically but this is what our end value time should be for the date-only case.